Matrix/Matrix/PrintMatrix.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Matrix
+{
+    /// <summary>
+    /// Implementation of print for matrices
+    /// </summary>
+    public class PrintMatrix
```

Comment:
```suggestion
    public static class PrintMatrix
```
Иначе кто-нибудь будет создавать объекты этого класса или ещё что-то такое